tests/Behat/Resources/services/contexts/ui.yml

```diff
@@ -5,6 +5,7 @@ services:
             - "@bitbag_sylius_quadpay_plugin.page.admin.payment_method.create"
         tags:
             - { name: fob.context_service }
+        public: true
```

Comment:
If you use `_defaults.public`, you'll not need to publish each service in this file.